### PR TITLE
Improved README to better accomodate the new change_hostname role

### PR DESCRIPTION
**Pull request by sthirugn**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,28 +34,20 @@ This project can be used to quickly setup Satellite 6.1 or 6.2 install with the
...
    Note: The ansible playbook run will fail and not proceed if the free space in root partition is less than the value specified in `required_root_free_space` variable in `roles/sat6repro/vars/main.yml`
 
-### Workstation requirements:
-3. Install `ansible` package in your workstation. For RHEL boxes, [access to EPEL] (https://access.redhat.com/solutions/3358) is required.
+#### On Workstation
 
-    ```console
-     # yum install -y ansible
-    ```
-4. git clone this project in your workstation.
```
Comment (johnpmitsch) [inline README.md]: should we say git clone this project on your Satellite? We are assuming they are running the playbooks on their satelllite in the next section, so changing what we refer to the machine as may cause some confusion

File: README.md [hunk 2]
```diff
@@ -1,15 +1,28 @@
...
+    ```console
+     # yum install -y ansible
+    ```
+- git clone this project in your workstation.
 
-### Pre-requisites:
+    ```console
+     # git clone git@github.com:RedHatSatellite/satellite-clone.git
+    ```
+- Now click one one of the links below to proceed further:
```
Comment (johnpmitsch) [inline README.md]: How about something like:

`Now you can proceed to any of the following tasks:`

File: README.md (hunk 1 above)
Comment (johnpmitsch) [inline README.md]: My reasonning is for the first link, they would just be clicking a link for something that is already right below

**Comment by sthirugn** [comment]
see https://github.com/sthirugn/satellite-clone/tree/fix_readme to have a preview of how the readme will look

File: README.md [hunk 3]
```diff
@@ -21,28 +34,20 @@ This project can be used to quickly setup Satellite 6.1 or 6.2 install with the
...
-    ```console
-     # git clone git@github.com:RedHatSatellite/satellite-clone.git
-    ```
-5. Download the data backup tar files - config, pgsql, mongodb in your workstation under the project folder - satellite-clone/roles/sat6repro/files so Ansible can find them.
-6. Update the required sections in `roles/sat6repro/vars/main.yml`.
-7. Update the `inventory` file under the root of this project to add your Satellite VM's ip address.
-8. Run the ansible playbook:
+1. Download the data backup tar files - config, pgsql, mongodb in your workstation under the project folder - satellite-clone/roles/sat6repro/files so Ansible can find them.
+2. Update the required sections in `roles/sat6repro/vars/main.yml`.
+3. Update the `inventory` file under the root of this project to add your Satellite VM's ip address. If executing the playbook on localhost, add `ansible_connection=local` after the IP address.
```
Comment (johnpmitsch) [inline README.md]: Can we say "Add the IP address of the blank machine you are cloning to"? The machine is not a Satellite machine yet.

**Comment by sthirugn** [comment]
Comments incorporated and more improvements added to make it user friendly.

**Comment by johnpmitsch** [comment]
looks really good, its much clearer now. ACK

**Pull request merged**